registry/garbage-collection.md

```diff
@@ -72,15 +72,15 @@ collection. Layer `a` had one reference removed but will not be garbage
 collected as it is still referenced by manifest `A`. The blob representing
 manifest `B` will also be eligible for garbage collection.
 
-After garbage collection has been run manifest `A` and its blobs remain.
+After garbage collection has been run, manifest `A` and its blobs remain.
 
 ```
 A -----> a
     \--> b
 ```
 
 
-## How Garbage Collection works
+## How Garbage Collection works?
```

Comment:
Is the question mark (?) unnecessary in heading "Why Garbage Collection?", too? 
Is the question mark (?) required only if the heading is a complete sentence, like "How does garbage collection work?"? 
A little confused.